### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by elisehuard**
add some minimal info to README + C* staging schema matching development environment's

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,21 @@ You will need [Leiningen][] 2.0.0 or above installed.
...
+
+Data Store: Cassandra
+
+Example Configuration in <https://github.com/MastodonC/witan.app/blob/master/resources/dev.witan-app.edn>: this file is also the default. If necessary, copy to home under .witan-app.edn and modify to suit.
+
 ## Running
 
 To start a web server for the application, run:
 
     lein ring server
```
Comment (acron0) [inline README.md]: I'd change this to `lein ring server-headless`

Comment (elisehuard) [inline README.md]: was actually default doc (from lein template :) )
for the beginning user it's not bad to have the docs opening automatically.

Comment (acron0) [inline README.md]: Ah, of course

**Comment by acron0** [comment]
:shipit:

**Pull request merged**